applications/schema/inputs.py

```diff
@@ -65,3 +92,43 @@ class WinterStorageApplicationInput(BaseApplicationInput):
     chosen_areas = graphene.List(
         graphene.NonNull(WinterStorageAreaChoiceInput), required=True
     )
+
+
+class UpdateBerthApplicationInput(BaseUpdateApplicationInput):
+    boat_draught = graphene.Float()
+    boat_weight = graphene.Float()
+    accessibility_required = graphene.Boolean()
+    boat_propulsion = graphene.String()
+    boat_hull_material = graphene.String()
+    boat_intended_use = graphene.String()
+    renting_period = graphene.String()
+    rent_from = graphene.String()
+    rent_till = graphene.String()
+    boat_is_inspected = graphene.Boolean()
+    boat_is_insured = graphene.Boolean()
+    agree_to_terms = graphene.Boolean()
```

Comment:
I first wrote "shouldn't this be required", but found out that it's not required on model level either. What happens if it is set false? :) In my opinion the "Agree to terms" could have been an UI only feature. Since it's been not mandatory earlier either, of course it's ok like this.

Reply:
Yeah, I'm actually not sure where is this required 😅 but since this is the `update` action, it's not mandatory, but then it probably has to be removed from the update because in theory it was accepted already.